python_modules/libraries/dagster-dbt/dagster_dbt/cli/resources_v2.py

```diff
@@ -705,6 +708,7 @@ def run(
         manifest: DbtManifest,
         project_dir: Path,
         target_path: Path,
+        raise_on_error: bool,
```

Comment:
I think we should default this to `True`, like we do for `execute_in_process` and `materialize`.

Reply:
We default to true in `.cli(...)`